src/components/Tabs/ContactComponent.tsx

```diff
@@ -52,6 +58,24 @@ export default function ContactComponent() {
     setMessage("");
     onClose();
   };
+  const onOpenFeatured = async () => {
```

Comment:
can this be changed to sendEmail() and onCloseFeatured() -> resetForm()? Unless there was a specific reason you named them this way